creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -705,14 +705,11 @@ class SnapbackSM {
         const walletsOnReplica = replicasToWalletsMap[replica]
 
         // Make requests in batches, sequentially, to ensure POST request body does not exceed max size
-        for (
-          let i = 0;
-          i < walletsOnReplica.length;
-          i += MAX_BATCH_CLOCK_STATUS_BATCH_SIZE
-        ) {
+        const maxBatchSize = config.get('maxBatchClockStatusBatchSize')
```

Comment:
nit - i think the appropriate name for this var is `batchSize` it doesnt actually matter in consumption that it is equivalent to the max value

Reply:
good point -- renamed!